### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by verdotte**

### CODE REVIEW

File: controllers/post.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import Post from '../models/post';
+import errors from '../helpers/errors';
+
+class post {
```
Comment (oesukam) [inline controllers/post.js]: @verdotte A class name should always start with a capital letter

File: controllers/post.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        title, description, poster, createdAt, updatedAt,
+      } = req.body;
+      const result = await Post.create({
+        title,
+        description,
+        poster,
+        createdAt,
+        updatedAt,
+      });
+      return res.status(200).json({
```
Comment (oesukam) [inline controllers/post.js]: @verdotte You can't have status different from what you are returning in the body

File: controllers/post.js (hunk 1 above)
Comment (verdotte) [reply controllers/post.js]: Okay

File: controllers/post.js (hunk 2 above)
Comment (verdotte) [reply controllers/post.js]: Cool 
Am going to fix it

**Comment by oesukam** [comment]
@verdotte Let me know when you are done implementing the feedback

**Comment by oesukam** [comment]
@verdotte You can also add the PR description

**Pull request merged**